agent/config/config.go

```diff
@@ -196,33 +196,33 @@ func (cfg *Config) trimWhitespace() {
 }
 
 func fileConfig() (Config, error) {
-	config_file := utils.DefaultIfBlank(os.Getenv("ECS_AGENT_CONFIG_FILE_PATH"), "/etc/ecs_container_agent/config.json")
-	config := Config{}
+	fileName := utils.DefaultIfBlank(os.Getenv("ECS_AGENT_CONFIG_FILE_PATH"), "/etc/ecs_container_agent/config.json")
```

Comment:
While you are here, can you const `"/etc/ecs_container_agent/config.json"` as well?

Reply:
Done